packages/terra-tabs/src/common-tabs/_HiddenTab.jsx

```diff
@@ -112,24 +127,34 @@ const HiddenTab = ({
     { 'is-disabled': isDisabled },
     theme.className,
   );
+  const tabDeleteLabel = `${intl.formatMessage({ id: 'Terra.tabs.hint.removable' })}`;
 
   const handleOnSelect = (event) => {
     event.preventDefault();
     event.stopPropagation();
 
     enableFocusStyles(event);
     onSelect(itemKey, metaData);
-    onChange(event, itemKey);
+    if (onChange) {
+      onChange(event, itemKey);
+    }
   };
 
   const onKeyDown = (event) => {
     if (event.nativeEvent.keyCode === KEY_RETURN || event.nativeEvent.keyCode === KEY_SPACE) {
       handleOnSelect(event);
-      onChange(event, itemKey);
+      if (onChange) {
+        onChange(event, itemKey);
+      }
     } else {
       handleArrows(event, index, tabIds);
     }
   };
+  function onCloseClick(event) {
+    event.stopPropagation();
+    // setIsClosed(true);
```

Comment:
This can be removed if not required

Reply:
Done,thanks https://github.com/cerner/terra-framework/commit/f4bd3b60b2862061fc633330bda690d094cd0ea6